services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -59,57 +76,120 @@ class Config(CommonConfig):
         env_prefix = "DIRECTOR_"
 
 
-class DynamicServicesSettings(BaseSettings):
-    enabled: bool = Field(True, description="Enables/Disables connection with service")
+_DYNAMIC_SIDECAR_DOCKER_IMAGE_RE = (
+    r"(^(local|itisfoundation)/)?(dynamic-sidecar):([\w]*)"
```

Comment:
please check the RE, I think at the end you should have a ```+``` not a ```*```

Reply:
you are correct, plus is what I wanted